GW2EIBuilders/Resources/htmlTemplates/tmplSimpleRotation.html

```diff
@@ -43,23 +43,43 @@
                 </li>
             </ul>
         </div>
+        <div class="d-flex flex-row justify-content-center mt-1 mb-1">
+            <ul class="nav nav-pills mr-1 ml-1 scale85">
+                <li class="nav-item rot-animfull">
+                    <a class="nav-link" @click="aftercastBorder = !aftercastBorder"
+                        :class="{active: aftercastBorder}" data-original-title="Toggle full after cast border">Full After Cast</a>
+                </li>
+                <li class="nav-item rot-cancelled">
+                    <a class="nav-link" @click="interruptedBorder = !interruptedBorder"
+                        :class="{active: interruptedBorder}" data-original-title="Toggle interrupted cast border">Interrupted</a>
+                </li>
+                <li class="nav-item rot-instant">
+                    <a class="nav-link" @click="instantBorder = !instantBorder"
+                        :class="{active: instantBorder}" data-original-title="Toggle instant cast border">Instant</a>
+                </li>
+                <li class="nav-item rot-unknown">
+                    <a class="nav-link" @click="unknownBorder = !unknownBorder"
+                        :class="{active: unknownBorder}" data-original-title="Toggle unknown cast border">Unknown</a>
+                </li>
+            </ul>
+        </div>
         <div style="z-index: 1;" class="mb-4 mt-2">
             <a href="#" class="dropdown-toggle" data-toggle="dropdown" title="Filter Skills" style="font-size: 1.1em;">
                 Skills <span class="caret"></span>
             </a>
             <ul class="dropdown-menu p-2 font-weight-normal scrollabledropdown" style="min-width: 165px; max-width: 165px;">
                 <div class="d-flex flex-column justify-content-center align-items-center mb-1"
-                    style="border-bottom: 2px solid #bbb;">
-                    <li>
-                        <button style="width:130px; padding: 0.4rem 0.4rem;" type="button" class="btn btn-primary btn-sm"
+                     style="border-bottom: 2px solid #bbb;">
```

Comment:
What is going on on this block? The indentation got all messed up.

Reply:
No idea either, it wasn't showing me any changes in vs, i'll try to revert all of them later tonight

Reply:
Hopefully now it should all be fine